tests/unit/test-cmd/test.run.js

```diff
@@ -187,6 +188,13 @@ describe('run', () => {
     assert.equal(reloadStrategy.called, false);
   });
 
+  it('Reload Delay', async () => {
+    const cmd = prepareRun();
+    const {reloadDelay} = cmd.argv;
+
+    assert.equal(typeof(reloadDelay), typeof(new Number()));
```

Comment:
This proposed test provides little value, because the type system already ensures that the `reloadDelay` parameter has the right type.

The unit test should verify that the functionality that you've added works as expected. A prerequisite to writing good unit tests is an understanding of the feature itself. A basic test should show that the feature implements the required functionality.
A good test is able to catch bugs in the implementation. To write a good test, try to think what could go wrong with the implementation, and keep that in mind while writing a test.